system-int-test/test/cmr/system_int_test/bootstrap/rebalance_collections_test.clj

```diff
@@ -47,10 +51,6 @@
        (is (= {:status 400
                :errors ["Provider: [NON_EXIST] does not exist in the system"]}
               (bootstrap/start-rebalance-collection "C1-NON_EXIST" {:synchronous false}))))
-     (testing "Non existent collection"
```

Comment:
This validation should be back in place so we should put this test back in (moving to separate index which is the default target should fail validation).